lib/parse.js

```diff
@@ -2236,17 +2236,86 @@ function parse($TEXT, options) {
         })
     }
 
+    function import_nameAsterisk() {
+        var start = S.token;
+        var foreign_name;
+        var name;
+
+        next();
+
+        var end = prev();
+
+        name = new AST_Symbol({
```

Comment:
I don't think `AST_Symbol` is designed to be used directly - you should always be instantiating one of its sub-classes.

If you need help determining which class to use, given me a hint as to what part of the statement this is trying to parse (and what that kind of statement does in ES6) and I can give you a hand.